app/javascript/src/components/SceneEditor.tsx

```diff
@@ -44,37 +40,46 @@ class SceneEditor extends React.Component<SceneEditorProps> {
           />
         </div>
 
-        <div className="SceneEditorField">
-          <label className="SceneEditorHeading" htmlFor="content">Content</label>
-          <textarea name="content" ref={this.editor} defaultValue={text} />
-        </div>
+        <SceneEditorTextAreaField
+          name="content"
+          title="Content"
+          defaultValue={text}
+          placeholderText=""
+          instructionalText=""
+          onChange={this.onChangeContent.bind(this)}
 
+        />
         <div className="SceneEditorField">
           <h3 className="SceneEditorHeading">Choices</h3>
 
           <ChoiceEditor focus={focus} requestPaint={requestPaint} />
-        </div>
+          </div>
+
+        <SceneEditorTextAreaField
+          name="notes"
+          title="Editor Notes"
+          placeholderText="Enter any editor-only notes you have here"
+          instructionalText="This box is for adding comments, new ideas, or general notes for this scene. These notes are not visible to the user."
+          defaultValue={notes}
+          onChange={this.onChangeNotes.bind(this)}
+        />
       </aside>
     )
   }
 
-  private install() {
-    if (this.editor.current) {
-      $R(this.editor.current, {
-        buttons: ['format', 'bold', 'italic', 'lists'],
-        callbacks: {
-          synced: (html: string) => this.onChange(html)
-        }
-      })
-    }
-  }
 
-  private onChange(html: string) {
+   private onChangeContent(html: string) {
```

Comment:
extra space character here as well

Reply:
this and above issue resolved in fa8add8